CMakeLists.txt

```
@@ -18,7 +18,9 @@ endif()
 include(CMakeDependentOption)
 include(TargetChompSources)
 
-find_package(AudioMan REQUIRED)
+if ("${CMAKE_GENERATOR_PLATFORM}" STREQUAL "Win32")
+  find_package(AudioMan REQUIRED)
+endif()
```

```suggestion
find_package(AudioMan)
```

Instead of per-platform checks, we can just do `find_package` without the `REQUIRED`.